jdsd_dsiii_practice_tool.toml

```diff
@@ -25,6 +25,7 @@ commands = [
     { flag = "hurtbox", hotkey = "f9" },
     { flag = "debug_draw", hotkey = "f9" },
     { flag = "evt_draw", hotkey = "f10" },
+    { flag = "bloodstain_draw", hotkey = "f11" },
```

Comment:
Considering it needs `debug_draw` like `hurtbox`, I'm thinking we could change `debug_draw` to be a different key :thinking:

Reply:
I removed the hotkeys for some of the debug draw options, since they seem unnecessary and we're kinda running out of f keys. Looks like I accidentally left position enabled in the config too, however I feel like that's a useful thing to have, so I left it there for now.